docs/release-notes/17-0-0/README.md

```diff
@@ -3,24 +3,255 @@ title: OpenProject 17.0.0
+
+#### Unified “My meetings” iCal subscription
+
+To avoid duplicate or confusing calendar invites, **users can now subscribe to all their meetings through a single iCal subscription URL** from the My meetings page or settings. External calendars (for example Outlook, Apple Calendar, or Open-Xchange) stay in sync automatically. Individual *.ics files* remain available when needed, but sending them is now more clearly controlled via dedicated options when creating or updating meetings.
+
+![My meetings page: Clicking on the More menu on the upper right corner opens a clickable option to "Subscribe to calendar"](openproject-17-0-meeting-subscribe-calendar.png)
+
+### Updated SharePoint integration with more restrictive permissions (Enterprise add-on)
+
+Before OpenProject 17.0, the Microsoft 365 file storage integration was a single combined OneDrive/SharePoint integration available as an Enterprise add-on in the Professional plan. With this release, it is now split into **two separate integrations** — one for **OneDrive** and one for **SharePoint** — giving administrators clearer setup options and more flexibility.
+
+For the SharePoint integration, OpenProject 17.0 introduces support for Microsoft’s *Sites.Selected* permission model. This allows administrators to grant the OpenProject Entra ID application access only to specific SharePoint sites, instead of using the broader *`Files.ReadWrite.All`* permission required in earlier versions.
+
+This new option helps organizations meet stricter security and compliance requirements while preserving all existing functionality.
+
+When configuring SharePoint storage, administrators now benefit from:
+
+- Separate, clearer setup options for OneDrive and SharePoint.
+- Support for the *Sites.Selected* permission scope, requiring a SharePoint Site ID.
+- Improved helper texts and documentation links.
+- Enhanced validation and error messages during configuration.
+
+OpenProject continues to support both the legacy permission model and the new *Sites.Selected* approach, allowing administrators to choose whichever fits their security standards.
+
+screenshot
+
+### Redesigned project overview with new tabs, configurable widgets, and an improved layout
+
+OpenProject 17.0 introduces a **major redesign of the project overview**, making key information easier to understand and share.
+
+The page is now split into two tabs:
+
+- **Overview** — a clean, structured summary with fixed widgets such as description, status, members, subprojects, news, lifecycle dates, and project attributes.
+- **Dashboard** — the fully editable area for custom widgets, now without the previous right-hand panel.
+
+The redesign also introduces **new widgets** (Budget, Sub-items) and improves existing ones through updated Primer styling. Administrators can additionally decide whether specific **project attribute sections** appear in the sidebar or directly in the main Overview as dedicated widgets.
```

Comment:
Budgets will only come with 17.1.
We also had a lenghty discussion about whether subitems should be written with or without a dash. In the end, we decided to go with the option without a dash.